test/tax-lot/tax-lot.repository.mock.ts

```diff
@@ -0,0 +1,12 @@
+import { generateMock } from "@anatine/zod-mock";
+import { findByBblSpatialSchema } from "src/tax-lot/tax-lot.repository.schema";
+
+export class TaxLotRepositoryMock {
+  findByBblSpatialMocks = Array.from(Array(10), () =>
+    generateMock(findByBblSpatialSchema, { seed: 1 }),
+  );
+
+  findByBblSpatial(bbl: string) {
+    return this.findByBblSpatialMocks.find((row) => row.bbl === bbl);
```

Comment:
Use basic code to emulate the behavior of the real method. 

Also, remember the point of mocking is to test the code that relies on the mocked class. We should avoid writing tests of the mocked class.